knowledge-base/hide-softkeyboard-without-losing-focus-maui-entry.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Hiding the SoftKeyboard without Losing Focus on the Entry
+description: Learn how to hide the SoftKeyboard without losing focus on the Entry control in MAUI.
+type: how-to
+page_title: How to Hide SoftKeyboard without Losing Focus on the MAUI Entry | Telerik
+slug: hide-softkeyboard-without-losing-focus-maui-entry
+tags: maui, entry, softkeyboard, hide, focus
+res_type: kb
+---
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 6.8.0 | Telerik UI for .NET MAUI Entry| [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 
+
+## Description
+
+This article describes how to hide the soft keyboard without loosing the focus on the Entry control.
+
+## Solution
+
+To hide the SoftKeyboard without losing focus on the Entry control in MAUI, you can use one of the following options:
```

Comment:
Add the two options as bullet lists here and configure them as references to the corresponding two sections. This will allow you to keep the current heading structure of the article. 

Note that you shouldn't introduce sections with an introductory sentence like this as it hurts the relative independence of each section.